topsites.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+import sys
+import time
+import datetime
+import hmac
+import hashlib
+import base64
+import collections
+import requests
+
+from urllib import urlencode
+from xml.dom.minidom import parseString
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import sessionmaker
+
+# Constant for alexa top site API
+ATS_ACTION_NAME = "TopSites"
+ATS_RESPONSE_GROUP_NAME = "Country"
+ATS_SERVICE_HOST = "ats.amazonaws.com"
+ATS_AWS_BASE_URL = "http://" + ATS_SERVICE_HOST + "/?"
+
+ATS_DATEFORMAT = "%Y-%m-%dT%H:%M:%S.{}Z"
+ATS_HASH_ALGORITHM = "HmacSHA256"
+
+# Regions/Countries to dump to topsites.db
+regions = {
+    "": (10000, 1),  # Global
```

Comment:
Understood. From the point of view of the python program, this is a dictionary. And the first part is called the key. Having an empty key doesn't seem to be a good choice. I think it's better to special case it in the call. with something like 

if regions['world']:
    parameter = ""